src/software/ai/hl/stp/tactic/shoot_goal_tactic.cpp

```diff
@@ -74,14 +75,11 @@ bool ShootGoalTactic::isEnemyAboutToStealBall() const
     Vector front_of_robot_dir =
         Vector(robot->orientation().cos(), robot->orientation().sin());
 
-    auto steal_ball_rect_width = DynamicParameters->getAIConfig()
-                                     ->getShootGoalTacticConfig()
-                                     ->EnemyAboutToStealBallRectangleWidth()
-                                     ->value();
-    auto steal_ball_rect_length = DynamicParameters->getAIConfig()
-                                      ->getShootGoalTacticConfig()
-                                      ->EnemyAboutToStealBallRectangleExtensionLength()
-                                      ->value();
+    auto steal_ball_rect_width =
+        shoot_goal_tactic_config->EnemyAboutToStealBallRectangleWidth()->value();
+    auto steal_ball_rect_length =
+        shoot_goal_tactic_config->EnemyAboutToStealBallRectangleExtensionLength()
```

Comment:
What is the extension length for?

Reply:
it's used to define the length of the rectangle in front of the robot where if an enemy robot is inside, we consider it a threat